BaseFilter/sections/foreign.txt

```diff
@@ -6580,6 +6580,8 @@ roweroweporady.pl#@#.ad_container
 @@||lib.wtg-ads.com/prebid/prebid_*.js$domain=tvn.pl
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/200976
+walbrzyszek.com##.kontener-reklam
```

Comment:
Remove the comment with the task link.

Reply:
It's not necessary for this, but if there was adb detection or incorrect blocking, you could add it.